packages/graphql-auth-transformer/src/ModelAuthTransformer.ts

```diff
@@ -205,7 +274,17 @@ export class ModelAuthTransformer extends Transformer {
             const argument = directive.arguments.find(get(arg))
             return argument ? valueFromASTUntyped(argument.value) : dflt
         }
-
+        let protectPrivateFields = true;
+
+        // get model args
+        const modelDirective = parent.directives.find((dir) => dir.name.value === 'model')
+        const parentModelArgs: ModelDirectiveArgs = modelDirective ? getDirectiveArguments(modelDirective) : {};
+        //  check if subscriptions is explicity disabled or if it's set to public
+        if ('subscriptions' in parentModelArgs &&
```

Comment:
The checks this way are not really explicit, based on a lot of JS behavior. What about making these checks more explicit? I can't see how are we checking for ```PUBLIC``` here? if subscriptions are explicitly disabled then it should have a ```null``` value.